public/wp-content/themes/ColoredCow/functions.php

```diff
@@ -67,22 +89,40 @@ function verify_credentials(){
         $meta_key = 'event_users';
         $event_users = get_post_meta( $event_id, $meta_key, true );
         $rsvp_date = date("d/m/y");
-        $rsvp_guest_id = $wpdb->get_var("SELECT post_id FROM $tablename WHERE meta_value = '".$_POST['guest_email']."'");
+        $guest_email = $_POST['guest_email'];
+        $rsvp_guest_id = $wpdb->get_var("SELECT post_id FROM $tablename WHERE meta_value = '".$guest_email."'");
         $hash = get_post_meta($rsvp_guest_id,'password',true);
-
+        $guest_name = get_the_title($rsvp_guest_id);
+        $soiree_name = get_the_title($event_id);
+        $soiree_date = get_post_meta($event_id,'event_date',true);
+        $host_info = get_userdata(get_current_user_id());
```

Comment:
This might not work if the current user doesn't have an account. Issue reported as testing feedback. It'll produce different results when current is signed/not signed.